test/integration/local/test_multi_model_endpoint.py

```diff
@@ -20,8 +20,8 @@
 import pytest
 import requests
 
-from multi_model_endpoint_test_utils import make_invocation_request, make_list_model_request, \
-    make_get_model_request, make_load_model_request, make_unload_model_request
+from multi_model_endpoint_test_utils import make_invocation_request, make_list_model_request,\
+    make_load_model_request, make_unload_model_request
```

Comment:
use multi-line import:

```suggestion
from multi_model_endpoint_test_utils import (
    make_invocation_request,
    make_list_model_request,
    make_load_model_request,
    make_unload_model_request,
)
```